opds.py

```diff
@@ -162,7 +162,9 @@ def annotate_work_entry(self, work, active_license_pool, edition,
                 title=unicode(group_title)
             )
 
-        # TODO: maybe we can do better than this in calculating updated()
+        # NOTE: This is a default that works in most cases. When
+        # ordering ElasticSearch results by last update time, you get
+        # a more reliable value that you can use instead.
```

Comment:
The code to use that is in circulation then?

Reply:
I think I should move this comment inside the `if` statement. The code for doing this is in core, it's just that search results are rarely sorted this way.

Reply:
Ah, I was confused because it didn't need to change in this PR.